.circleci/config.yml

```diff
@@ -378,6 +378,20 @@ commands:
                 key: "<< pipeline.parameters.merge_version >>-lint"
                 paths:
                   - target
+  xtask_release_preverify:
+    steps:
+      - restore_cache:
+          keys:
+            - "<< pipeline.parameters.merge_version >>-preverify"
+      - run: xtask release pre-verify
+      - when:
+          condition:
+            equal: [ "dev", "<< pipeline.git.branch >>" ]
```

Comment:
This isn't intuitive to me.  Is this the right conditional branch we want to check?  "When the current pipeline's branch is `dev` do this?"  How about releases that happen off a tag on a release branch?

Reply:
Ahh, we can just remove the stuff around caching.

Reply:
[a85f195](https://github.com/apollographql/router/pull/3859/commits/a85f195794e5bd29b50f4d4e46d80a50da770a94)